rpm/felix.spec

```diff
@@ -2,7 +2,7 @@
 
 Name:           felix
 Summary:        Project Calico virtual networking for cloud data centers
-Version:        2.2.1
+Version:        2.4.0
```

Comment:
Erk.  The fact this jumped from 2.2.1 to 2.4.0 - did we miss a change for 2.3.0?